src/main/kotlin/app/revanced/patches/youtube/misc/playertype/PlayerTypeHookPatch.kt

```diff
@@ -1,25 +1,27 @@
-package app.revanced.patches.youtube.misc.playertype.patch
+package app.revanced.patches.youtube.misc.playertype
 
 import app.revanced.extensions.exception
-import app.revanced.patcher.annotation.Description
-import app.revanced.patcher.annotation.Name
 import app.revanced.patcher.data.BytecodeContext
 import app.revanced.patcher.extensions.InstructionExtensions.addInstruction
 import app.revanced.patcher.extensions.InstructionExtensions.addInstructions
 import app.revanced.patcher.extensions.InstructionExtensions.getInstruction
 import app.revanced.patcher.patch.BytecodePatch
-import app.revanced.patcher.patch.annotations.DependsOn
-import app.revanced.patches.youtube.misc.integrations.patch.IntegrationsPatch
+import app.revanced.patcher.patch.annotation.Patch
+import app.revanced.patches.youtube.misc.integrations.IntegrationsPatch
 import app.revanced.patches.youtube.misc.playertype.fingerprint.PlayerTypeFingerprint
 import app.revanced.patches.youtube.misc.playertype.fingerprint.VideoStateFingerprint
 import com.android.tools.smali.dexlib2.iface.instruction.ReferenceInstruction
 
-@Name("Player type hook")
-@Description("Hook to get the current player type and video playback state.")
-@DependsOn([IntegrationsPatch::class])
-class PlayerTypeHookPatch : BytecodePatch(
-    listOf(PlayerTypeFingerprint, VideoStateFingerprint)
+@Patch(
+    description = "Hook to get the current player type and video playback state.",
+    dependencies = [IntegrationsPatch::class]
+)
```

Comment:
❓ : Missing patch name

```suggestion
@Patch(
    name = "Player type hook",
    description = "Hook to get the current player type and video playback state.",
    dependencies = [IntegrationsPatch::class]
)
```